### PR TITLE
added containers for the body and weather boxes

### PR DESCRIPTION
**Pull request by 7KAG7**

### CODE REVIEW

File: app/javascript/react/components/FootwearContainer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react"
+
+const FootwearContainer = props => {
+
+  return (
+    <div className="body-boxes grid-x medium-6">
+
```
Comment (prophetsfall) [inline app/javascript/react/components/FootwearContainer.js]: Unnecessary extra line.

File: app/javascript/react/components/App.js [hunk 2]
```diff
@@ -1,7 +1,16 @@
 import React from 'react'
+import { Route, Switch, BrowserRouter } from "react-router-dom"
+
```
[the remark this answers is not in the document]
Comment (7KAG7) [reply app/javascript/react/components/App.js]: fixed, I thought if it was standard react related items and component imports you separated them out.

File: app/javascript/react/components/FootwearContainer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      </h6>
+
+      <h6 htmlFor="head-box-2-label" className="box-label">Footwear Accessory 2:
+      <box type="text" className="head-box"></box>
+      </h6>
+
+      <h6 htmlFor="head-box-3-label" className="box-label">Footwear Accessory 3:
+      <box type="text" className="head-box"></box>
+      </h6>
+
```
[the remark this answers is not in the document]
Comment (7KAG7) [reply app/javascript/react/components/FootwearContainer.js]: corrected

File: app/javascript/react/components/FootwearContainer.js (hunk 1 above)
Comment (7KAG7) [reply app/javascript/react/components/FootwearContainer.js]: corrected

File: app/javascript/react/components/WeatherBoxContainer.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className="primary-weather-box grid-x medium-6">
+      <h4 htmlFor="primary-weather" id="weather-box-title">Todays Weather:
+      <box type="text" id="primary-weather-box">
+      
+      </box>
+      </h4>
+
+
+
```
[the remark this answers is not in the document]
Comment (7KAG7) [reply app/javascript/react/components/WeatherBoxContainer.js]: corrected

**Pull request merged**